src/style.css

```
@@ -0,0 +1,132 @@
+* {
+  box-sizing: border-box;
+}
+
+.home-page {
+  font-family: Roboto, sans-serif;
+  --color-text-1: #060b35;
+  --color-text-2: #616070;
+}
+
+.home-page__content {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__name {
+  line-height: 18px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: var(--color-text-1);
+  margin: 0 16px 4px;
+}
+
+.card__code {
+  line-height: 14px;
+  width: 98px;
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-text-2);
+  margin: 0 84px 16px 16px;
+}
+
+.card__block-1 {
+  margin: 0 16px 24px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__block-2 {
+  margin: 0 16px 16px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  width: 96px;
+  height: 16px;
+}
+
+.stars-star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--1 .stars-star:nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}
+
+.card__text {
+  margin: 0;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-text-1);
+}
+
+.card__price {
+  line-height: 18px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  color: var( --color-text-2);
+  margin: 0;
+}
+
+.card__price-num {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color-text-1);
+  margin: 0;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 700;
+  color: #fff;
+
+  margin: 0 16px 16px;
+  width: 166px;
```

```suggestion
  width: 100%;
```
will work after you apply paddings to the card